### ISSUE TITLE
Empty transaction basket

### ISSUE THREAD

**Issue by tranduyhung** [hipay/hipay-fullservice-sdk-php#64]
When I get a transaction via the PHP SDK, it doesn't include the basket.

```
$transactionId = A_TRANSACTION_REFERENCE_HERE;

$config = new Configuration([
    //...
]);

$clientProvider = new SimpleHTTPClient($config);

$gatewayClient = new GatewayClient($clientProvider);

$transaction = $gatewayClient->requestTransactionInformation($transactionId);

$basket = $transaction->getBasket(); // $basket is empty.
```

If we make direct API call to get the transaction, the basket info is available.

```
$transactionId = A_TRANSACTION_REFERENCE_HERE;

$url = "https://stage-secure-gateway.hipay-tpp.com/rest/v1/transaction/$transactionId";

$headers = [
    //...
];

$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, $url);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
curl_setopt($ch, CURLOPT_POST, 1);
$response = curl_exec($ch);
curl_close($ch);

$response = json_decode($response);

print_r($response);
```

We get the basket:

```
stdClass Object
(
    [transaction] => stdClass Object
        (
            ...
        )
    [basket] => stdClass Object
        (
            [item] => stdClass Object
                (
                    [type] => good
                    [productReference] => ...
                    [quantity] => ...
                    [unitPrice] => ...
                    [name] => ...
                    [totalAmount] => ...
                )

        )

)
```

**Comment by JBaptisteLvt** [comment]
Hello !
Thank you for this issue !
There is indeed an issue with how the basket is populated in the PHP SDK. We will add the fix to our roadmap.

**Comment by JBaptisteLvt** [comment]
hello !
This has been released in 2.13.0 !
